internal/handlers/common/error.go

```diff
@@ -32,12 +32,14 @@ const (
 	// For ProtocolError only.
 	errInternalError = ErrorCode(1) // InternalError
 
-	ErrBadValue          = ErrorCode(2)     // BadValue
-	ErrNamespaceNotFound = ErrorCode(26)    // NamespaceNotFound
-	ErrNamespaceExists   = ErrorCode(48)    // NamespaceExists
-	ErrCommandNotFound   = ErrorCode(59)    // CommandNotFound
-	ErrNotImplemented    = ErrorCode(238)   // NotImplemented
-	ErrRegexOptions      = ErrorCode(51075) // Location51075
+	ErrBadValue                       = ErrorCode(2)     // BadValue
+	ErrNamespaceNotFound              = ErrorCode(26)    // NamespaceNotFound
+	ErrNamespaceExists                = ErrorCode(48)    // NamespaceExists
+	ErrCommandNotFound                = ErrorCode(59)    // CommandNotFound
+	ErrNotImplemented                 = ErrorCode(238)   // NotImplemented
+	ErrProjectionInclusionInExclusion = ErrorCode(31253) // Cannot do inclusion on field array in exclusion projection
+	ErrProjectionExclusionInInclusion = ErrorCode(31253) // Cannot do exclusion on field array in inclusion projection
```

Comment:
There are several problems:
* the error code for that one is 31254;
* "array" there is a field name, not a part of the the error message;
* that comment specifies a string value produced by `stringer` tool (that wasn't run), not a error message.